app/src/main/java/ru/prometeydev/movie/ui/moviesdetails/MoviesDetailsNavigable.kt

```diff
@@ -0,0 +1,17 @@
+package ru.prometeydev.movie.ui.moviesdetails
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import ru.prometeydev.movie.ViewModelProviderFactory
+import ru.prometeydev.movie.common.popBack
+import ru.prometeydev.movie.common.toast
+
+open class MoviesDetailsNavigable : Fragment() {
```

Comment:
I don't understand idea about this base class

Reply:
I wanted to do some kind of abstraction, but it turned out not beautiful, probably I will remove it, I will correct